src/select.coffee

```diff
@@ -213,6 +213,10 @@ Ember.AddeparMixins.ResizeHandlerMixin,
 
   contentProxy: Ember.computed ->
     optionLabelPath = @get('optionLabelPath')
+    if optionLabelPath is ''
```

Comment:
it's `''` by default but there's nothing stopping someone from setting it to one of those values